apps/mobile/src/components/secure-your-wallet/skip-secure-wallet-modal.tsx

```diff
@@ -0,0 +1,50 @@
+import { RefObject } from 'react';
+import { useSharedValue } from 'react-native-reanimated';
+
+import Cross from '@/assets/cross-large.svg';
+import { BottomSheetModal } from '@gorhom/bottom-sheet';
+import { useTheme } from '@shopify/restyle';
+
+import { Box, Theme, TouchableOpacity } from '@leather.io/ui/native';
+
+import { CLOSED_ANIMATED_POSITION, Modal } from '../bottom-sheet-modal';
+import { Button } from '../button';
+import { TransText } from '../trans-text';
+
+interface SkipSecureWalletModalProps {
+  skipSecureWalletModalRef: RefObject<BottomSheetModal>;
+  onSkip(): unknown;
```

Comment:
```suggestion
  onSkip(): void;
```

more common typing here if returning nothing?

Reply:
Sure, not particularly fussed either way. Claude has this to say:

_In Typescript, what's the difference between `callback(): void` and `callback(): unknown`_
```
Usage Scenarios
void:

Use void when you do not expect a callback to provide any return value and you are only interested in the side effects of the function.
Suitable for event handlers, logging functions, or any situation where the return value is not needed.
unknown:

Use unknown when you expect the callback to return some kind of value, but you don't know the specific type at the time of defining the callback.
Suitable for situations where functions can return various types of results, and you need to handle each case safely with type checks.
```